scenarios/happypath/build/build_application.go

```diff
@@ -107,7 +107,33 @@ func Application(ctx context.Context, cfg config.Config) error {
 		}
 	}
 
+	pipelineRuns := job.GetPipelineRuns(ctx, cfg, defaults.App2Name, jobName)
+	for _, run := range pipelineRuns {
+		var targetTask *models.PipelineRunTask
+		tasks := job.GetPipelineRunTasks(ctx, cfg, defaults.App2Name, jobName, *run.RealName)
+		for _, t := range tasks {
```

Comment:
Recommend to use method [FindFirst](https://github.com/equinor/radix-common/blob/cabe99c6819cfd4b549e8ead9cad3eff355eea70/utils/slice/utils.go#L75) for readability